pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -281,7 +282,7 @@
   /**
    * The application name (require server version &gt;= 9.0)
    */
-  APPLICATION_NAME("ApplicationName", null, "name of the application (backend >= 9.0)"),
+  APPLICATION_NAME("ApplicationName", DriverInfo.DRIVER_NAME, "Name of the Application (backend >= 9.0)"),
```

Comment:
Theoretically, one might want a connection without this property. I don't have a real world example for that, just being cautious about unnecessary changes :-)